truncate_client/src/web_comms.rs

```diff
@@ -0,0 +1,56 @@
+use eframe::web_sys;
+use futures::channel::mpsc::{Receiver, Sender};
+use futures::SinkExt;
+use futures_util::{future, pin_mut, StreamExt};
+use truncate_core::messages::{GameMessage, PlayerMessage};
+use ws_stream_wasm::{WsMessage, WsMeta};
+
+pub async fn connect(
+    connect_addr: String,
+    tx_game: Sender<GameMessage>,
+    rx_player: Receiver<PlayerMessage>,
+) {
+    use web_sys::console;
+
+    console::log_1(&format!("Connecting to {connect_addr}").into());
+
+    let (_ws, wsio) = WsMeta::connect(connect_addr, None)
+        .await
+        .expect("assume the connection succeeds");
```

Comment:
Conventional use of `expect` would be `expect("Connection failed") or something, right?

Reply:
Depends on who you ask 😉 I don't think there's a strong consensus either way.

First, NB: this isn't a great usage of `expect`. That message doesn't add anything so it should just be `unwrap`
Second, NB: this error should be handled, this is just a temp expect because I'm lazy.

But _in general_, I don't think `expect("Connection failed")` adds any value as a message. That error will already likely be printed, so you'll get something roughly like:

```
Error: "Connection failed" 
Error occurred unwrapping Err("Failed connection")
```

I generally try to write them in a `should` phrasing. So more like:

```rust
let (_ws, wsio) = WsMeta::connect(connect_addr, None)
        .await
        .expect("the socket address should be valid and reachable");
```

aka providing the reasoning for _why_ I expect the value to be true. (This helps point out in this case that this is bad `expect`, since I haven't done anything to validate that assumption). 

With that, your error message prints the error, and the reason you expected there to _not_ be an error, which is far more useful.

Final nitpick: It reads way better as a sentence. `expect("Connection failed")`... am I expecting the connection to fail?